src/Auth.ts

```diff
@@ -13,6 +13,8 @@ import config from './config.js';
 import request from './request.js';
 import { settingsNames } from './settingsNames.js';
 import { browserUtil } from './utils/browserUtil.js';
+import * as accessTokenUtil from './utils/accessToken.js';
+import { IdentityDetails } from './m365/commands/IdentityDetails.js';
 
```

Comment:
In the FileTokenStorage, let's rename connectionInfoFilePath  to `.cli-m365-connection.json` and then introduce another one for all connections named `.cli-m365-all-connections.json`